lib/transforms/addPrecacheServiceWorker.js

```diff
@@ -214,7 +214,7 @@ module.exports = (
     } else {
       batches = entryPointHtmlAssets.map(asset => [asset]);
     }
-    await Promise.map(batches, generateAndAttachPrecacheServiceWorkerAsync, {
+    await Bluebird.map(batches, generateAndAttachPrecacheServiceWorkerAsync, {
```

Comment:
https://github.com/sindresorhus/p-map is a drop-in replacement.

Reply:
Done in 4fd02081